mteb/tasks/Clustering/zho/CMTEBClustering.py

```diff
@@ -1,11 +1,124 @@
 from __future__ import annotations
 
+import itertools
+
+from datasets import Dataset, DatasetDict
+
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.AbsTaskClusteringFast import AbsTaskClusteringFast
 from mteb.abstasks.TaskMetadata import TaskMetadata
 
-from ....abstasks.AbsTaskClustering import AbsTaskClustering
+NUM_SAMPLES = 2048
+
+
+class CLSClusteringFastS2S(AbsTaskClusteringFast):
+    metadata = TaskMetadata(
+        name="CLSClusteringFastS2S",
```

Comment:
```suggestion
        name="CLSClusteringS2S.v2",
```
To keep it consistent with other changes.

Reply:
I just realized that `v2` was not added to my other conversions. Should we add that in https://github.com/embeddings-benchmark/mteb/issues/660? cc @x-tabdeveloping

Reply:
Mine either actually (though I converted most things to Hierarchical). I think we should prefer versioning over overwriting, but what's merged is merged

Reply:
I'm confused about what is being overwritten here. And it's no problem - as we want to be consistent - they can be changed to align with others. 

If we want v2 then the guide needs updating as well. I personally have no preference. Could @KennethEnevoldsen @x-tabdeveloping you guys decide pleeeease?

Reply:
I prefer the `.v2` personally, I think we should roll with that.

Reply:
Thanks, I'll make the changes 👍